### PR TITLE
Add artifacts for opensearch cli 1.1

### LINKED ISSUE #557: Publish new release for opensearch-cli in OpenSearch Download Page
**Is your feature request related to a problem? Please describe.**
NA

**Describe the solution you'd like**
We would like to update opensearch-cli download link to latest version

**Describe alternatives you've considered**
N/A

**Additional context**
Relevant issue to release artifacts: https://github.com/opensearch-project/opensearch-build/issues/833

### PR DESCRIPTION
**Pull request by VijayanB**
Signed-off-by: Vijayan Balasubramanian <balasvij@amazon.com>

### Description
Add markdown and update latest version to 1.1
 
### Issues Resolved
#557 
### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _artifacts/opensearch-cli/opensearch-cli-1.1.0-linux-x64.markdown [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+role: cli
+artifact_id: opensearch-cli
+version: 1.1.0
+platform: linux
+architecture: x64
+artifact_url: htpps://artifacts.opensearch.org/opensearch-clients/opensearch-cli/opensearch-cli-1.1.0-linux-x64.zip
```
Comment (stockholmux) [inline _artifacts/opensearch-cli/opensearch-cli-1.1.0-linux-x64.markdown:7]: `htpps`?

File: _versions/2021-06-07-opensearch-1.0.0-rc1.markdown [hunk 2]
```diff
@@ -13,7 +13,7 @@ components:
   -
     role: ui
     artifact: opensearch-dashboards
-    version: 1.0.0-rc1
+    version: 1.1.0-rc1
```
Comment (stockholmux) [inline _versions/2021-06-07-opensearch-1.0.0-rc1.markdown:16]: I'm not sure why you're changing this one - this line is causing an error (and it's opensearch dashboard not CLI)

**Review by stockholmux — CHANGES_REQUESTED** [review]
Two small issues.

File: _versions/2021-06-07-opensearch-1.0.0-rc1.markdown (hunk 2 above)
Comment (VijayanB) [reply _versions/2021-06-07-opensearch-1.0.0-rc1.markdown]: Ack.

File: _artifacts/opensearch-cli/opensearch-cli-1.1.0-linux-x64.markdown (hunk 1 above)
Comment (VijayanB) [reply _artifacts/opensearch-cli/opensearch-cli-1.1.0-linux-x64.markdown]: Ack.

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**